Merged current color schemes to 'profiles.json'.

doterik opened the pull request:
I found it simpler to first merge schemes for testing of different colors and then delete unwanted ones.

If you are interested...

mbadolato commented:
Hi, can you provide a little more detail? What would this be used for, and how? 

Thanks

doterik commented:
Hi,

I had very little idea of how the different color schemes looked like. Some I recognized since before but most of them I did not know how they looked.

I did a template (the PR) of all schemes and copied them into the real profiles.json. Then I removed the '//', line by line, saved the profile, and could see the colors live in the terminal when I moved along.

For me, that worked well, but maybe not a very good idea to publish like this. I have no problem at all to reject this.

mbadolato commented:
Gotcha, thanks for the invite. For future reference, you can go into the`tools/` directory and use the preview script in there!

```bash
$ cd tools
$ ./preview.rb ../schemes/*
```
Then hit the spacebar and you'll cycle through each of the themes. Or instead of using `*` list just the schemes you want to preview

Cheers!
